testfiles/CarrollMRT-RefBldgLargeOfficeNew2004_Chicago.idf

```diff
@@ -95,7 +96,7 @@
     6;                       !- Minimum Number of Warmup Days
 
   PerformancePrecisionTradeoffs,
-    Yes,                      !- Use Coil Direct Solutions
+    No,                       !- Use Coil Direct Solutions
```

Comment:
Noticed that the CarrollMRT idf had the coil direct solution active. Turned this off per @nealkruis